package.json

```diff
@@ -51,5 +56,17 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
+  },
+  "babel": {
```

Comment:
Can we add this to a babelrc file for consistency?

Reply:
This has the same purpose as `.babelrc` (it's shared between both webpack and jest), but I can extract the config anyway if you prefer.

Reply:
We're using .*rc files for our other tooling, and I think it's best to be consistent when possible.